admin/spec/service/check-window.service.spec.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-/* global describe expect it beforeEach afterEach */
+/* global describe expect it beforeEach afterEach xdescribe xit */
```

Comment:
these are suffering from the promise rejection issue we discussed?

Reply:
Yes - there is some work required here.  Not caused by this branch, but discovered when I was investigating all stray unit test output.